thefuck/shells/nushell.py

```diff
@@ -0,0 +1,72 @@
+from subprocess import Popen, PIPE
+import os
+import six
+import sys
+from .. import logs
+from ..utils import DEVNULL
+from .generic import Generic
+
+
+class Nushell(Generic):
+    friendly_name = 'Nushell Shell'
+
+    def get_aliases(self):
+        aliases = {}
+        command = 'help aliases | select name expansion | each { |row| $row.name + " ; " + $row.expansion } | str join (char nl)'
+        proc = Popen(['nu', '-l', '-c', command], stdout=PIPE, stderr=DEVNULL)
+        if proc.stdout is None:
+            return aliases
+        alias_out = proc.stdout.read().decode('utf-8').strip()
+        for alias in alias_out.split('\n'):
+            split_alias = alias.split(" ; ")
+            if len(split_alias) == 2:
+                name, value = split_alias
+                aliases[name] = value
+        return aliases
```

Comment:
Would it be possible to get all the commands in nushell aswell using `help commands` in here?
`help aliases` only gets the actual aliases, i.e, stuff like `alias vim = nvim`.

Reply:
good catch, I'll change that

Reply:
sweet, this will be awesome when its working merged!